frontend/src/context/IndexerDatabaseProvider.jsx.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { createContext, useEffect, useCallback, useState } from "react";
```

Comment:
Let's have it in a separate PR. I don't understand why we cannot use the old provider and just subscribe to a different topic based on the configuration option. Please, make sure the backend sends the data of the same shape, so we can just have a thin tweak to the `DatabaseProvider` changing the WAMP topic